lit-element/src/HelloWorld.js

```diff
@@ -0,0 +1,21 @@
+import { LitElement, html, css } from 'lit-element';
```

Comment:
if one day esLint rules get added to the project, this file can have linter issues for declared variables and not use them. E.g **css** is not being used anywhere.

Reply:
Yep, css again :smile: Done.